package.json

```diff
@@ -47,6 +47,7 @@
     "babel-jest": "26.3.0",
     "jest": "26.4.2",
     "jest-transform-stub": "2.0.0",
+    "msw": "0.21.2",
```

Comment:
Antar denne er tenkt brukt for testing? Kunne vi brukt `/pages/api` til dette og?

Reply:
Den kan brukes til test og som mock for utvikling. Både testing-library og msw tenker litt "nytt" omkring test og mock og slikt :-)